spec/ruby/core/io/each_byte_spec.rb

```diff
@@ -15,6 +15,16 @@
     -> { IOSpecs.closed_io.each_byte {} }.should raise_error(IOError)
   end
 
+  ruby_bug '#17661', ''...'3.0.1' do
```

Comment:
Can you please update this from 3.0.1 to 3.1? It's unwise to expect backports to specific point versions.  If there is a backport, we can always adjust the spec guard at that point.